android/src/main/java/io/mosip/tuvali/common/safeExecute/TryExecuteSync.kt

```diff
@@ -12,7 +13,7 @@ class TryExecuteSync(private val bleExceptionHandler: OpenIdBLEExceptionHandler)
       try {
         returnValue = fn()
       } catch (e: Exception) {
-        bleExceptionHandler.handleException(Exception("Unknown Exception", e))
+        bleExceptionHandler.handleException(UnknownException("Caught unknown exception in Try Execute", e))
```

Comment:
I think, the exception can be sent as is to the bleExceptionHandler and let it classify exception type based on where it is originated.